feature-utils/poly-look/test/react-components/chip.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import Chip from "../../src/react-components/chip.jsx";
+
+const chipId = "chipId";
+const mockedHandleClick = jest.fn();
+
+const chip = <Chip id={chipId} handleClick={mockedHandleClick} active />;
+
+it("Chip can be clicked", () => {
+  const { container, getByText } = render(chip);
+  expect(container).toBeTruthy();
+  fireEvent.click(getByText(chipId), mockedHandleClick);
+  expect(mockedHandleClick).toHaveBeenCalled();
+});
+
+it("Chip has been selected", () => {
+  const { getByText } = render(chip);
+  const chipElement = getByText(chipId);
+  if (chip.props.active) {
+    expect(chip.props.active).toBeTruthy();
+    expect(chipElement).toHaveClass("chip selected");
+  } else {
+    expect(chip.props.active).toBeFalsy();
+    expect(chipElement).not.toHaveClass("chip selected");
+  }
```

Comment:
I think that what's tested is consistency, mainly, between property and class. I would, however, add maybe an additional test for the class that's actually supposed to have if it's inactive.

Reply:
> I think that what's tested is consistency, mainly, between property and class. I would, however, add maybe an additional test for the class that's actually supposed to have if it's inactive.

There isn't a class when it's inactive, we are just adding or removing the class "selected".

Reply:
Well, it will have "chip" by itself, right? WRT to that, you can probably remove them. I simply used them initially to get the feel of the assertions.